website/src/ServiceEndpoint.jsx

```diff
@@ -3,5 +3,6 @@
  */
 
 export default {
-  host: 'comakeserver.herokuapp.com'
+  //host: 'comakeserver.herokuapp.com'
```

Comment:
Will we need this ("comakeserver.herokuapp.com") in the future?  If not, I think it should be removed.